code/game/jobs/job/captain.dm

```diff
@@ -3,7 +3,7 @@ var/datum/announcement/minor/captain_announcement = new(do_newscast = 1)
 /datum/job/captain
 	title = "Captain"
 	flag = CAPTAIN
-	department = "Command"
+	department = DEPARTMENT_COMMAND
 	head_position = 1
```

Comment:
So we have two completely separate ways of telling if someone is a head?

1. If they're in `positions_by_department[DEPARTMENT_COMMAND]`
2. If they have `head_position = TRUE`

One needs to go - Or both do & turn things into flags like I suggested above.

Reply:
Someone is a head for the department set in `department` if they have `head_position = true`. Technically you can _tell_ that someone is a head by looking into DEPARTMENT_COMMAND, but that will not necessarily be true in the future, and if that check is used somewhere it should be gone, you're right.

I'll fix the manifest code that does this.

Reply:
Manifest code rewritten. I could just remove `positions_by_department` since it's not needed anymore.